test/unit/deadline_submitter_for_blender/test_submitter.py

```diff
@@ -368,49 +381,72 @@ def test_get_param_values(submitter_settings, common_layer_settings):
     }
     expected = [{"name": k, "value": v} for k, v in expected_settings.items()]
 
-    filled = template_filling.get_parameter_values(
-        submitter_settings, common_layer_settings, queue_params=[]
-    )
-    assert filled == expected
+    # Patching out the scene cycles device setting will not add the CyclesRenderDevice parameter
+    with patch("bpy.context.scene.cycles.device", "CPU"):
```

Comment:
I know that this was already the state of these tests. But I worry over time this will balloon in size. What do you think about either breaking this out into distinct tests or turning this into a parameterized test? This will make it easier to reason about test failures and we'll set ourselves up well for the future.